draft-ietf-rtcweb-jsep.xml

```diff
@@ -1977,13 +1977,265 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
+            stop and an error MUST be returned.</t>
+            <t>Next, the SessionDescription is parsed into a data structure,
+              as described in the
+              <xref target="sec.parsing-a-desc"/> section below. If parsing
+              fails for any reason, processing MUST stop and an error MUST be
+              returned.
+            </t>
+            <t>Finally, the parsed SessionDescription is applied as described in
+              the <xref target="sec.applying-a-local-desc"/> section below.</t>
+          </list>
+        </t>
+      </section>
+
+      <section title="Processing a Remote Description" anchor="sec.processing-a-remote-desc">
+        <t>When a SessionDescription is supplied to setRemoteDescription, the
+            following steps MUST be performed:
+          <list style="symbols">
+            <t>First, the type of the SessionDescription is checked against the
+                  current state of the PeerConnection:
+              <list style="symbols">
+                <t>If the type is "offer", the PeerConnection state MUST be
+                      either "stable" or "have-remote-offer".</t>
+                <t>If the type is "pranswer" or "answer", the PeerConnection
+                      state MUST be either "have-local-offer" or
+                      "have-remote-pranswer".</t>
+              </list>
+            </t>
+            <t>If the type is not correct for the current state, processing MUST
+            stop and an error MUST be returned.</t>
+            <t>Next, the SessionDescription is parsed into a data structure,
+              as described in the
+              <xref target="sec.parsing-a-desc"/> section below. If parsing
+              fails for any reason, processing MUST stop and an error MUST be
+              returned.
+            </t>
+            <t>Finally, the parsed SessionDescription is applied as described in
+              the <xref target="sec.applying-a-remote-desc"/> section below.</t>
+          </list>
+        </t>
+      </section>
+
+      <section title="Parsing a Session Description" anchor="sec.parsing-a-desc">
+        <t>When a SessionDescription of any type is supplied to
+        setLocal/RemoteDescription, the implementation must parse it and
+        reject it if it is invalid. The exact details of this
+        process are explained below.</t>
+        <t>The SDP contained in the session description object consists of a
+        sequence of text lines, each containing a key-value expression,
+        as described in <xref target="RFC4566"/>, Section 5.
+        The SDP is read,
+        line-by-line, and converted to a data structure that contains the
+        deserialized information. However, SDP allows many types of lines, not
+        all of which are relevant to JSEP applications, and the ones that are
+        not specifically referenced below MAY be ignored. Implementations MAY
```

Comment:
I don't think that can work. We spell out exactly what has to happen in createOffer, I think we have to do the same here. Yes, new things will be added in the future, and we need to at a minimum not choke on such things, but for the things we know about I don't think we can just say "consult the relevant RFCs" since there is so much ambiguity already.